src/NuGet.Core/NuGet.Packaging/Rules/BestPractice/MissingReadmeRule.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Common;
+
+
+namespace NuGet.Packaging.Rules
```

Comment:
By default Visual Studio will put new files in a namespace that includes project sub-directories as part of the namespace, now that this file was moved, it's no longer following that convention.  NuGet does this often, but personally I really don't like it. Both directories and namespaces are used for organization, so if it's worthwhile organizing a file, then I think it's also worthwhile organizing the namespace.

Was this done intentionally, or just a mistake since the file was moved, rather than created from the new subdirectory?